bundle/Form/FieldTypeHandler/Checkbox.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\InformationCollectionBundle\Form\FieldTypeHandler;
+
+use Ibexa\Contracts\Core\FieldType\Value;
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+use Ibexa\Contracts\Core\Repository\Values\ContentType\FieldDefinition;
+use Ibexa\Core\FieldType\Checkbox as CheckboxValue;
+use Ibexa\Core\Helper\FieldHelper;
+use Netgen\Bundle\InformationCollectionBundle\Form\FieldTypeHandler;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+final class Checkbox extends FieldTypeHandler
+{
+    protected FieldHelper $fieldHelper;
+
+    public function __construct(FieldHelper $fieldHelper)
+    {
+        $this->fieldHelper = $fieldHelper;
+    }
+
+    public function convertFieldValueToForm(Value $value, ?FieldDefinition $fieldDefinition = null): bool
+    {
+        return $value->bool;
```

Comment:
Inline-hint Checkbox Value.

Reply:
Fixed in b540b6f5

Reply:
Fix aliasing `Checkbox` type as `CheckboxValue` and use FQCN in inline type hint (elsewhere as well).

Reply:
solved in f5744574